docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md

```diff
@@ -38,155 +38,81 @@ dropdowns below.
-  <tags />
-  <description>Attach transcription generated by the AmberScript service.
-    This is an internal workflow, started by the Transcription Service.
-  </description>
-  <configuration_panel />
-
-  <operations>
-
-    <!-- Attach caption/transcript -->
-
-    <operation id="amberscript-attach-transcription"
-      fail-on-error="true"
-      exception-handler-workflow="partial-error"
-      description="Attach captions/transcription">
-      <configurations>
-        <!-- This is filled out by the transcription service when starting this workflow -->
-        <configuration key="transcription-job-id">${transcriptionJobId}</configuration>
-        <!-- Caption generated will have the default flavor based on the target-caption-format e.g. captions/vtt -->
-        <configuration key="target-caption-format">vtt</configuration>
-        <configuration key="target-tag">engage-download</configuration>
-      </configurations>
-    </operation>
-
-    <!-- Merge caption/transcript to existing publication and republish -->
-    <operation id="publish-engage"
-      fail-on-error="true"
-      exception-handler-workflow="partial-error"
-      description="Distribute and publish to engage server">
-      <configurations>
-        <configuration key="download-source-flavors">dublincore/*,security/*</configuration>
-        <configuration key="download-source-tags">engage-download</configuration>
-        <configuration key="strategy">merge</configuration>
-        <configuration key="check-availability">false</configuration>
-      </configurations>
-    </operation>
-
-    <operation
-      id="republish-oaipmh"
-      exception-handler-workflow="partial-error"
-      description="Update recording metadata in default OAI-PMH repository">
-      <configurations>
-        <configuration key="source-flavors">dublincore/*,security/*</configuration>
-        <configuration key="download-source-tags">engage-download</configuration>
-        <configuration key="repository">default</configuration>
-      </configurations>
-    </operation>
-
-    <!-- Archive media package -->
-
-    <operation id="snapshot"
-      fail-on-error="true"
-      exception-handler-workflow="partial-error"
-      description="Archive media package">
-      <configurations>
-        <configuration key="source-flavors">*/*</configuration>
-      </configurations>
-    </operation>
-
-    <!-- Clean up work artifacts -->
-
-    <operation
-        id="cleanup"
-        fail-on-error="false"
-        description="Remove temporary processing artifacts">
-      <configurations>
-        <configuration key="delete-external">true</configuration>
-        <!-- FixMe Don't clean up ACLs until workflow service no longer looks for them in the WFR. -->
-        <configuration key="preserve-flavors">security/*</configuration>
-      </configurations>
-    </operation>
-
-  </operations>
-
-</definition>
+<summary>amberscript-start-transcription.yaml</summary>
```

Comment:
@Arnei this is `amberscript-start-transcription`, as is the one below.  This one was originally `amberscript-attach-transcription` - so our docs are missing the attach part of the workflows

Reply:
🤦. Thanks for catching that.